packages/oft-evm/test/mocks/SendUln302Mock.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
```

Comment:
I think this one we already have in `packages/test-devtools-evm-foundry/contracts/mocks/SendUln302Mock.sol` so you can add an NPM dependency and import

Reply:
Wish I could;  that one only supports OpenZeppelin v5.  OFT uses OpenZeppelin V4.